CMakeLists.txt

```diff
@@ -92,9 +92,9 @@ endif ()
 # Note: this must come after `project(...)` otherwise
 # `CMAKE_CXX_COMPILER_ID` is undefined.
 if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    set(IS_GCC_OR_CLANG TRUE)
+    set(_is_gcc_or_clang TRUE)
```

Comment:
[fix] `IS_GCC_OR_CLANG` is used in multiple cmake files, so I think should be uppercased.